analytics/management/commands/active_user_stats_by_day.py

```diff
@@ -19,7 +20,7 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         # type: (*Any, **Any) -> None
         if options["date"] is None:
-            date = datetime.datetime.now() - datetime.timedelta(days=1)
+            date = timezone.now() - timezone.timedelta(days=1)
```

Comment:
I don't see a `timezone.timedelta` in the docs, though I didn't look too carefully. If there isn't a `timezone.timedelta`, the `datetime.timedelta` is correct.